docs/cli.md

```diff
@@ -1302,7 +1330,7 @@ Info: Successfully registered auth application
   "secret": "supersupersecretsecret"
 }
 
-viam auth-app update --org-id=z1234567-1a23-45a6-a11b-abcdefg1234 --application-id=1234a1z9-ab2c-1234-5678-bcd12345678a --redirect-uris="https://test.com","https://test2.com"
+viam auth-app update --org-id=z1234567-1a23-45a6-a11b-abcdefg1234 --application-name="julias app" --application-id=1234a1z9-ab2c-1234-5678-bcd12345678a --redirect-uris="https://test.com","https://test2.com"
```

Comment:
So auth-app also has a `get` command now. Did you check all the commands on what sub commands they now support?
```
➜  viam-docs git:(DOCS-2789/capture-sync-image-data) ✗ viam auth-app
NAME:
   viam auth-app - manage third party auth applications

USAGE:
   viam auth-app command [command options] [arguments...]

COMMANDS:
   register  register a third party auth application
   update    update a third party auth application
   get       get configuration for a third party auth application
   help, h   Shows a list of commands or help for one command
```

Reply:
`viam train` also has a bunch of new command options. Can you please go through them and add the missing ones?

Reply:
Yes, sorry I did not realize this was in scope. Will do

Reply:
done